### PR TITLE
[Order] introduce paymentTotal Property to store the rounded payment value with a precision of 2

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  |no
| BC breaks?    | no
| Deprecations? | no

### CODE REVIEW

File: src/CoreShop/Bundle/ThemeBundle/Service/InheritanceLocator.php [hunk 1]
```diff
@@ -75,6 +75,10 @@ protected function getPathsForAppResource($parameters)
 
     protected function resolveHierarchy()
     {
+        if (!$this->lastTheme) {
```
Comment (dpfaffenbauer) [inline src/CoreShop/Bundle/ThemeBundle/Service/InheritanceLocator.php:78]: should not be there, will be removed

**Comment by dpfaffenbauer** [comment]
Todo: add migration

File: src/CoreShop/Bundle/FrontendBundle/Resources/views/Common/order_detail.html.twig [hunk 2]
```diff
@@ -227,7 +227,7 @@
                 </td>
                 <td colspan="1"
                     class="text-right cart-total-price">
-                    {{ currency.format(order.getTotal, order.getCurrency.isoCode) }}
+                    {{ currency.format(order.getTotal, order.getCurrency.isoCode, 4) }}
```
Comment (solverat) [inline src/CoreShop/Bundle/FrontendBundle/Resources/views/Common/order_detail.html.twig:230]: remove the `4`.

Comment (dpfaffenbauer) [reply src/CoreShop/Bundle/FrontendBundle/Resources/views/Common/order_detail.html.twig]: done

**Pull request merged**